run-tests.sh

```diff
@@ -66,6 +66,8 @@ $(basename $0) options:
     -X        | xfstests git repo. Used by tests/xfstests.sh.
     -x        | xfstests git branch to checkout and track.
     -y        | xfstests ./check additional args
+    -z <file> | Specify the filesystem's meta data device path that contains
```

Comment:
-d and -z rubs me the wrong way.. maybe rename -D to -F for dumping ftrace on oops, and use -M for metadata and -D for data?  These single char options aren't looking like the best idea I've ever had.